test/extended/util/framework.go

```diff
@@ -1691,15 +1691,16 @@ type podExecutor struct {
 
 // NewPodExecutor returns an executor capable of running commands in a Pod.
 func NewPodExecutor(oc *CLI, name, image string) (*podExecutor, error) {
-	out, err := oc.Run("run").Args(name, "--labels", "name="+name, "--image", image, "--restart", "Never", "--command", "--", "/bin/bash", "-c", "sleep infinity").Output()
-	if err != nil {
-		return nil, fmt.Errorf("error: %v\n(%s)", err, out)
-	}
-	_, err = WaitForPods(oc.KubeClient().CoreV1().Pods(oc.Namespace()), ParseLabelsOrDie("name="+name), CheckPodIsReady, 1, 3*time.Minute)
-	if err != nil {
-		return nil, err
-	}
-	return &podExecutor{client: oc, podName: name}, nil
+	execPod := e2epod.CreateExecPodOrFail(oc.AdminKubeClient(), oc.Namespace(), name, func(pod *corev1.Pod) {
+		pod.Name = name
+		pod.GenerateName = ""
+		pod.Spec.Containers[0].Image = utilimage.ShellImage()
```

Comment:
Is hard-coding the image pullspec a problem for tests that don't currently use this image pullspec?  In particular, this test specifies a different image pullspec: https://github.com/openshift/origin/blob/f825ad788c95df1b805c89cbdfe912559bf575c2/test/extended/security/scc.go#L362-L371

Reply:
Yes it is - I mentioned that in a previous comment. Reverting back to custom imagespec.